### ISSUE TITLE
Another Azure Platform upgrade

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#682]
Chris reported `develop` CI failed this morning due to scheduled Azure brownout.  Looks like we need update the ubuntu images.

```
ubuntu-18.04 Python37


##[error]This is a scheduled Ubuntu-18.04 brownout. The Ubuntu-18.04 environment is deprecated and will be removed on December 1st, 2022. For more details, see https://github.com/actions/runner-images/issues/6002
,##[error]The remote provider was unable to process the request.
Pool: [Azure Pipelines](https://dev.azure.com/ComputationalCryoEM/1455a1f0-08c2-49c8-b076-83cebead3498/_settings/agentqueues?poolId=&queueId=26)
Image: ubuntu-18.04
```

My only concern is their naming is goofy...  They list 22.04, but still have 20.04 as `-latest`.  Makes me wonder if we should just always use specific versions. Maybe docs old. TBD.

https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml

**Comment by janden** [comment]
Could just be that by `-latest` they want something like `-stable`. 22.04 is less than six months old, so it's likely not as well deployed as 20.04 (see, e.g., https://fr.archive.ubuntu.com/stats/stats_of_day-25.html?version=last). I'd say we're better off testing for 20.04 rather than 22.04.

**Comment by janden** [comment]
Nm looks like we test for both. Great!

**Comment by garrettwrong** [comment]
Neat stat site. Yes, we test both.

I would prefer if Azure provided image tags as `-lastest` and `-stable` (and that they were deployed coherently), but I haven't seen anything to indicate that in the Azure docs... only `-latest` which actually seems to be current right now :).

**Comment by garrettwrong** [comment]
Closed by #683 .
